src/test/java/com/easypost/BetaRateTest.java

```diff
@@ -34,7 +51,11 @@ public static void setup() throws EasyPostException {
      */
     @Test
     public void testRetrieveStatelessRates() throws EasyPostException {
-        vcr.setUpTest("retrieve_stateless_rates");
+        List<MockRequest> mockRequests = new ArrayList<>();
+        mockRequests.add(new MockRequest(new MockRequestMatchRules(Requestor.RequestMethod.POST, ".*\\/rates.*"),
+                new MockResponse(200, MOCK_STATELESS_RATE_RESPONSE)));
+
+        vcr.setUpTest("retrieve_stateless_rates", mockRequests);
```

Comment:
We are not testing the internal function `lowest_rate` here, we should hit the API endpoint and shouldn't really mock this test